api/v1alpha1/nodequotaconfig_types.go

```diff
@@ -26,25 +26,44 @@ import (
 
 // NodeQuotaConfigSpec defines the desired state of NodeQuotaConfig
 type NodeQuotaConfigSpec struct {
-	ReservedHoursTolive int                  `json:"reservedHoursToLive,omitempty"`
-	ControlledResources []string             `json:"controlledResources,omitempty"`
-	Roots               []SubnamespacesRoots `json:"subnamespacesRoots,omitempty"`
+	// ReservedHoursTolive defines how many hours the reservedresources can live until they removed from the cluster resources
+	ReservedHoursTolive int `json:"reservedHoursToLive"`
```

Comment:
// ReservedHoursToLive defines how many hours the ReservedResources can live until they are removed from the cluster resources

Also rename `ReservedHoursTolive` to `ReservedHoursToLive`